### PR TITLE
Refactor/110/feedback changes testcases

### PR DESCRIPTION
**Pull request by rutgervincken**

### CODE REVIEW

**Comment by rutgervincken** [comment]
Linde doet dit morgen

**Comment by LvanWerven** [comment]
Matteo gaat deze nakijken

File: PatternPal/PatternPal.Tests/New_TestCasesRecognizers/Strategy/StrategyTestCase12.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Text;
 
 namespace PatternPal.Tests.TestClasses.Strategy
```
Comment (ViciousDoormat) [inline PatternPal/PatternPal.Tests/New_TestCasesRecognizers/Strategy/StrategyTestCase12.cs:5]: This implementation does not correspond to the explanation above.
The concrete strategy does not implement the interface anymore, yet in the explanation, there is a check at "is an implementation of the strategy interface".
Please also check that the other implementations do correspond with their explanation. Just to be sure.

**Review by ViciousDoormat — CHANGES_REQUESTED** [review]

**Comment by rutgervincken** [comment]
doe accepteren

**Review by JustNireon — CHANGES_REQUESTED** [review]
LGTM

**Review by JustNireon — APPROVED** [review]

**Pull request merged**